### PR TITLE
Update Octokit

### PR DESCRIPTION
**Pull request by gbiellem**
Updating Octokit 0.4 to Octokit 0.16 to fix problems with attaching assets to releases https://github.com/octokit/octokit.net/pull/895

### CODE REVIEW

**Comment by gbiellem** [comment]
@SzymonPobiega can you review this?  At the moment when releasing stuff the assets have to added manually which is a PITA. So keen to get this out.

File: src/Compiler/ReleaseManager.cs [hunk 1]
```diff
@@ -43,25 +43,24 @@ public ReleaseManager(GitHubClient gitHubClient, string organization)
...
 
                         var allIssues = await gitHubClient.AllIssuesForMilestone(milestone);
 
-                        var latestIssueModification =
-                            allIssues.Where(i => i.State == ItemState.Closed).Max(i => i.ClosedAt.Value).UtcDateTime;
-
-                        Console.Out.WriteLine("Release exists for milestone {0} - Last updated at: {1}, Issues updated at:{2}", potentialRelease, releaseUpdatedAt, latestIssueModification);
-
-                        if (releaseUpdatedAt < latestIssueModification)
+                        var allClosedIssues = allIssues.Where(i => i.State == ItemState.Closed).ToList();
```
Comment (SimonCropp) [inline src/Compiler/ReleaseManager.cs]: this will get gradually slower over time. better to do a sorted query so we dont need the `.max`

Comment (SzymonPobiega) [inline src/Compiler/ReleaseManager.cs]: Compared to the old version, it does not seem to make things worse.

**Comment by SzymonPobiega** [comment]
Looks OK to me.

**Pull request merged**